en-US/mozorg/technology.lang

```diff
@@ -0,0 +1,116 @@
+## Note: Demo server is available at https://www-dev.allizom.org/technology
+## URL: https://www-dev.allizom.org/%LOCALE%/technology
+
+;We’re opening up a whole new Internet.
+We’re opening up a whole new Internet.
+
+
+;Explore some of our latest innovations — built with open Web technologies and designed to help keep the Internet healthy and accessible forever.
+Explore some of our latest innovations — built with open Web technologies and designed to help keep the Internet healthy and accessible forever.
+
+
+# This  string is a potential navigation label
```

Comment:
What does this comment mean? Label for navigation should be in main.lang, not in this file.
I would drop the "potential".